### ISSUE TITLE
broken

### ISSUE THREAD

**Issue by Lumabots** [jay3332/pilmoji#39]
Traceback (most recent call last):
  File "/Users/luma/Code/V2/test.py", line 1, in <module>
    from pilmoji import Pilmoji
  File "/Users/luma/Code/V2/.venv/lib/python3.11/site-packages/pilmoji/__init__.py", line 1, in <module>
    from . import helpers, source
  File "/Users/luma/Code/V2/.venv/lib/python3.11/site-packages/pilmoji/helpers.py", line 18, in <module>
    language_pack: Dict[str, str] = unicode_codes.get_emoji_unicode_dict('en')
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'emoji.unicode_codes' has no attribute 'get_emoji_unicode_dict'

**Comment by annihilatorrrr** [comment]
@jay3332 please fix it fast

**Comment by Lumabots** [comment]
just downgrad emoji, i can fix his code with a fork but it will not be publish before a long time. Its better if u just download emoji, (pilmoji use emoji)
pip install emoji==2.11.0 should work

**Comment by Lumabots** [comment]
i made up a fix on my github, feel free to use that fork because idk when jay will merge my fix

**Comment by kxzk** [comment]
### quick fix

fork the repo and update the `requirements.txt`:

```python
Pillow
emoji==2.11.0
```

### longer fix

he removed `get_emoji_unicode_dict()` and placed it under `utils/testutils.py` - [here](https://github.com/carpedm20/emoji/blob/master/utils/testutils.py). likely just port over code below into `helpers.py` here.

```python
import emoji.unicode_codes

# ... omitted

_EMOJI_UNICODE: Dict[str, Any] = {lang: None for lang in emoji.LANGUAGES}  # Cache for the language dicts
_ALIASES_UNICODE: Dict[str, str] = {}  # Cache for the aliases dict

def get_emoji_unicode_dict(lang: str) -> Dict[str, Any]:
    """Generate dict containing all fully-qualified and component emoji name for a language
    The dict is only generated once per language and then cached in _EMOJI_UNICODE[lang]"""

    if _EMOJI_UNICODE[lang] is None:
        _EMOJI_UNICODE[lang] = {data[lang]: emj for emj, data in emoji.EMOJI_DATA.items()
                                if lang in data and data['status'] <= emoji.STATUS['fully_qualified']}

    return _EMOJI_UNICODE[lang]
```

**Comment by Lumabots** [comment]
> ### quick fix
> fork the repo and update the `requirements.txt`:
> 
> ```python
> Pillow
> emoji==2.11.0
> ```
> 
> ### longer fix
> he removed `get_emoji_unicode_dict()` and placed it under `utils/testutils.py` - [here](https://github.com/carpedm20/emoji/blob/master/utils/testutils.py). likely just port over code below into `helpers.py` here.
> 
> ```python
> import emoji.unicode_codes
> 
> # ... omitted
> 
> _EMOJI_UNICODE: Dict[str, Any] = {lang: None for lang in emoji.LANGUAGES}  # Cache for the language dicts
> _ALIASES_UNICODE: Dict[str, str] = {}  # Cache for the aliases dict
> 
> def get_emoji_unicode_dict(lang: str) -> Dict[str, Any]:
>     """Generate dict containing all fully-qualified and component emoji name for a language
>     The dict is only generated once per language and then cached in _EMOJI_UNICODE[lang]"""
> 
>     if _EMOJI_UNICODE[lang] is None:
>         _EMOJI_UNICODE[lang] = {data[lang]: emj for emj, data in emoji.EMOJI_DATA.items()
>                                 if lang in data and data['status'] <= emoji.STATUS['fully_qualified']}
> 
>     return _EMOJI_UNICODE[lang]
> ```

i already made up a fix and made a pu ll request but im not sure he will push it before a long time

**Comment by jay3332** [comment]
fixed in #40
